features/place/description.feature

```diff
@@ -49,3 +49,25 @@ Feature: Test place description property
     """
     ""
     """
+
+  Scenario: Delete the last description of a place
+    When I send a DELETE request to "%{placeUrl}/description/nl"
+    Then the response status should be "204"
+    And I send a GET request to "%{placeUrl}"
+    Then the response status should be "200"
+    And the JSON response should not have "description"
+
+  Scenario: Delete a description of a place, with one description left
+    When I set the JSON request payload to:
+    """
+    { "description": "French" }
+    """
+    And I send a PUT request to "%{placeUrl}/description/fr"
+    When I send a DELETE request to "%{placeUrl}/description/nl"
```

Comment:
This does nothing because of the empty initial description.

Reply:
I now create the descriptions with a put request